src/main/resources/application.yaml

```diff
@@ -75,17 +75,22 @@ ftp:
   downtime:
     from: ${FTP_DOWNTIME_FROM:16:00}
     to: ${FTP_DOWNTIME_TO:17:00}
-  service-folders:
+  services-config:
```

Comment:
perhaps, for clarity, this config could be moved out of the `ftp:` tree in the yaml (`displayName` has nothing to do with ftp)

Reply:
@rkondratowicz I'll close this pr and create another pr to add config under `reports:`